chia/wallet/wallet_user_store.py

```diff
@@ -65,12 +65,15 @@ async def create_wallet(
                 (id, name, wallet_type, data),
             )
             await cursor.close()
+            wallet = await self.get_last_wallet()
+            if wallet is None:
+                raise ValueError("Failed to get the just-created wallet")
```

Comment:
I went with `ValueError` because that was the specific error used in other handling of this situation here.  Not sure it's the best choice but aiming towards retaining behavior.

Reply:
yeah that makes sense to me